core/src/main/java/hudson/slaves/OfflineCause.java

```diff
@@ -132,4 +132,13 @@ public ByCLI(String message) {
             this.message = message;
         }
     }
+
+    /**
+     * Caused by idle period
```

Comment:
Please add `@since TODO` so merger will not forget to properly document new API. Also, the javadoc description should have trailing `.`.